src/credentials/vue/root.vue

```diff
@@ -1,15 +1,97 @@
 <template>
     <div class="flex-container" style="display: flex">
-        <div id="left-column">I am left column</div>
-        <div id="right-column">I am right column</div>
+        <div id="left-column">
+            <div>
+                <h1>Select a feature to get started</h1>
+                <div v-for="itemId in unlockedItemIds">
+                    <BaseServiceItem
+                        :title="getServiceItemProps(itemId).title"
+                        :description="getServiceItemProps(itemId).description"
+                        :status="'UNLOCKED'"
+                        :isSelected="isServiceSelected(itemId)"
+                        :id="itemId"
+                        :key="buildServiceItemKey(itemId, 'UNLOCKED')"
+                        @service-item-clicked="serviceWasSelected(itemId)"
+                    >
+                    </BaseServiceItem>
+                </div>
+            </div>
+
+            <div>
+                <h3>UNLOCK ADDITIONAL FEATURES</h3>
+                <div>Some features have additional authentication requirements to use. <a>Read more.</a></div>
+
+                <div v-for="itemId in lockedItemIds">
+                    <BaseServiceItem
+                        :title="getServiceItemProps(itemId).title"
+                        :description="getServiceItemProps(itemId).description"
+                        :status="'LOCKED'"
+                        :isSelected="isServiceSelected(itemId)"
+                        :id="itemId"
+                        :key="buildServiceItemKey(itemId, 'LOCKED')"
+                        @service-item-clicked="serviceWasSelected(itemId)"
+                    >
+                    </BaseServiceItem>
+                </div>
+            </div>
+            <h3></h3>
+        </div>
+        <div id="right-column"></div>
     </div>
 </template>
 
 <script lang="ts">
 import { defineComponent } from 'vue'
+import {
+    ServiceItemsState,
+    ServiceItemId,
+    ServiceStatus,
+    BaseServiceItem,
+    StaticServiceItemProps,
+} from './ServiceItem.vue'
 
 export default defineComponent({
-    name: 'MyComponent',
+    components: { BaseServiceItem },
+    name: 'AuthRoot',
+    data() {
+        return {
+            unlockedItemIds: [] as ServiceItemId[],
+            lockedItemIds: [] as ServiceItemId[],
+            serviceItemsState: new ServiceItemsState(),
```

Comment:
Classes are not easily serialized/de-serialized. I'd be wary of using them with `data`